libexec/rtld-elf/aarch64/rtld_machdep.h

```diff
@@ -178,17 +178,18 @@ make_data_cap(const Elf_Sym *def, const struct Struct_Obj_Entry *defobj)
 	(((uintptr_t (*)(uint64_t, uint64_t, uint64_t, uint64_t, uint64_t, \
 	    uint64_t, uint64_t, uint64_t))ptr)(0, 0, 0, 0, 0, 0, 0, 0))
 
+#define	TLS_TCB_SIZE	(2 * sizeof(void *))
```

Comment:
Technically doesn't have to move given calculate_first_tls_offset is a macro and not expanded before it's defined (calculate_tls_post_size already references it, too), but this makes more sense, and is what's done on riscv. Ideally TLS_VARIANT_I (added upstream since this point in time) and TLS_DTV_OFFSET would be hoisted too as on riscv, but meh, we don't have diffs to that for aarch64. Maybe something to tidy up upstream if we can be bothered. Or not given it should all go in a machine/tls.h at some point anyway, could just be cleaned up as part of that.

Reply:
I had already thought about perhaps cleaning this up upstream.  The change to use TLS_TCB_SIZE instead of the bare 16 is certainly worth upstreaming.  I might also just work on <machine/tls.h> in upstream as well.

Reply:
I was thinking it but felt I shouldn't say it :)